src/app/components/ParaTimePicker/index.tsx

```diff
@@ -36,15 +38,26 @@ const ParaTimePickerContent: FC<ParaTimePickerContentProps> = ({ onClose, onConf
   const { t } = useTranslation()
   const { network, layer } = useRequiredScopeParam()
   const [showNetworkMenu, setShowNetworkMenu] = useState(network !== Network.mainnet)
-  const [selectedLayer, setSelectedLayer] = useState<undefined | Layer>()
-  const [selectedNetwork, setSelectedNetwork] = useState<undefined | Network>(network)
-  const [hoveredLayer, setHoveredLayer] = useState<undefined | Layer>()
-  const [hoveredNetwork, setHoveredNetwork] = useState<undefined | Network>()
+  const [selectedLayer, setSelectedLayer] = useState<Layer>(layer)
+  const [selectedNetwork, setSelectedNetwork] = useState<Network>(network)
+  const selectNetwork = (newNetwork: Network) => {
+    setSelectedNetwork(newNetwork)
+    setSelectedLayer(RouteUtils.getEnabledLayersForNetwork(network)[0])
```

Comment:
- https://683bae47.oasis-explorer.pages.dev/mainnet/emerald
- switch to testnet
- shows "Emerald Testnet"
- click Select
- crashes, unsupported_layer

Fix is probably just
```suggestion
    setSelectedLayer(RouteUtils.getEnabledLayersForNetwork(newNetwork)[0])
```

Reply:
Yes that was exactly the original intention.